avatars/animationHelpers.js

```diff
@@ -1233,7 +1233,7 @@ export const _applyAnimation = (avatar, now, moveFactors) => {
       let defaultAnimation = 'grab_forward';
 
       const activateAction = localPlayer.getAction('activate');
-      if (activateAction.animationName) {
+      if (activateAction && activateAction.animationName) {
```

Comment:
Why would there be no `activateAction` but there is `activateTime`? This sounds like some code fails to clean up.

Reply:
It crashes, this patches it but doesn't address root. Will look into it.